packages/app-mobile/components/NoteBodyViewer/hooks/useOnMessage.ts

```diff
@@ -49,6 +52,14 @@ export default function useOnMessage(
 			onResourceLongPress(msg);
 		} else if (msg.startsWith('edit:')) {
 			onRequestEditResource?.(msg);
+		} else if (msg.startsWith('onscroll:')) {
```

Comment:
A `console.info` statement just above this logs a large amount of data because of the `onscroll:` event.

Because of this, it might make sense to add an exception above to skip logging on scroll events.

Reply:
Done in 30867472f28fb278e6330419df21623d1d67c6aa